src/index.js

```diff
@@ -1,3 +1,5 @@
+/* istanbul ignore file */
+
 import 'react-app-polyfill/ie11';
```

Comment:
this may not be necessary anymore?

Reply:
Quite true, thank God! Would just our testing in place along with manual testing suffice for the removal?

Reply:
@arcepaul Removed.